protocol/avdl/stellar1/local.avdl

```diff
@@ -131,6 +131,110 @@ protocol local {
   string getWalletAccountPublicKeyLocal(int sessionID, AccountID accountID);
   SecretKey getWalletAccountSecretKeyLocal(int sessionID, AccountID accountID);
 
+  // Get the non-XLM assets for sending.
+  array<SendAssetChoice> getSendAssetChoicesLocal(
+    int sessionID,
+    AccountID from,
+    string to // Optional. Username, acctID, or assertion.
+  );
+  record SendAssetChoiceLocal {
+    Asset asset;
+    // Enable the row. True means the recipient accepts this asset.
+    // False if there's no recipient, or an error loading them, or they don't accept.
+    boolean enabled;
+    string left; // Example: "BTC"
+    string right; // Examples: "Stronghold.com", "GBSTRH4QOTWNSVA6E4HFERETX4ZLSR3CIUBLK7AXYII277PFJC4BBYOG"
+    string subtext; // Examples: "", "max doesn't accept HUGZ"
+  }
+
+  // Check but do not send a payment.
+  // Opt  - Optional field.
+  // Opt* - Optional for this RPC but required for res.readyToSend to be set.
+  BuildPaymentResLocal buildPaymentLocal(
+    AccountID from,
+    // Opt. Stellar seqno of the sender when the user started the form.
+    // If it is old, the payment will be blocked blocked.
+    string fromSeqno,
+    // Opt*. Recipient username, acctID, or assertion.
+    // Examples: "", "russel", "GBQTE2V7Y356TFBZL6YZ2PA3KIILNSAAQRV5C7MVWS22KQTS4EMK7I46"
+    string to,                                    
+    bool toIsAccountID,
```

Comment:
should this be some kind of enum instead

Reply:
Username and assertion go mostly through the same code paths. Probably only these two will be used anyhow. Should be easy to change